### DISCUSSION TITLE
g-f works well in Neovim to open the file whose name is under the cursor.  How could that work as well in VSCode Neovim?

### DISCUSSION THREAD

**Discussion by ahjun001** [vscode-neovim/vscode-neovim#1236] [Q&A] [answered]
g-f works well in Neovim to open the file whose name is under the cursor.  
How could that work as well in VSCode Neovim?

I see that g-f is linked to some VS Code commands, see here:
https://github.com/vscode-neovim/vscode-neovim#vscode-specific-bindings

These might prevent the Neovim go-to-file (under cursor) behavior,
or how could this be fixed?

Thank you.

**Comment by martin-braun** [comment, marked as answer]
VSCode doesn't have a "go to file under cursor", but can install [Open file](https://marketplace.visualstudio.com/items?itemName=Fr43nk.seito-openfile) and map `gf` to `<CMD>call VSCodeCall('seito-openfile.openFileFromText')<CR>` in normal mode.

[I did exactly that in my Vim configuration](https://github.com/martin-braun/CocChad/commit/469b575845e8a2d4520c135ef733c56526e7fca2) and it works like charm.

**Comment by ahjun001** [comment]
Thank you @martin-braun for your reply and suggestion.

I take your word that it works but could not get it to work exactly this way.

From the links you provided and other files in your repos, it looks as if your binding was in a mappings.lua file, which I don't have in my machine.

What  worked for me was to install the VSCode 'Open file'  pluggin.  This create a Alt-P keybinding to OpenFile under Cursor.
I just edited the keystrokes to G F to get it to work like vanilla nvim.

Thank you again, definitely increasing productivity and happiness at work here!
